client/src/reducers/main.js

```diff
@@ -1,4 +1,5 @@
 import { combineReducers } from 'redux';
+import galleryReducer from './galleryReducer';
 import metaDataReducer from './metaDataReducer';
```

Comment:
I see the reducer add into the root reducer, however there isnt a galleryReducer file included in the commit.
Is it missing?